ask-sdk-core/.eslintrc.json

```diff
@@ -0,0 +1,248 @@
+{
+    "extends": [
+        "eslint:recommended"
```

Comment:
How did you get the eslint configuration and why are we not using airbnb?

Reply:
Currently all eslintrc are specific to project subdir ask-sdk-core/ , ask-sdk-controls/ since not all codebase have consistent rules.